### PR TITLE
Create button components.

### PR DESCRIPTION
**Pull request by agalcalledjen**

### CODE REVIEW

File: client/App.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Platform, StyleSheet, Text, View } from 'react-native';
+
+import Card from './js/screens/Card';
+// import RootStackNavigator from './navigation/RootStackNavigator';
+
+const instructions = Platform.select({
```
Comment (siddharthparmar7) [inline client/App.js:7]: please remove unnecessary code and imported modules.

File: client/js/screens/Login/styles.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import { StyleSheet } from 'react-native';
-import { container } from '../../config/styles';
+import { container, colors, padding } from '../../config/styles';
 
 const styles = StyleSheet.create({
-  container: { ...container }
+  container: { ...container },
+  button: {
+    width: 125
```
Comment (siddharthparmar7) [inline client/js/screens/Login/styles.js]: use flexbox instead of giving fix widths and heights to show your component the same size on all phones.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you!

File: client/App.js (hunk 1 above)
Comment (agalcalledjen) [reply client/App.js:7]: I have removed the unnecessary code and imported modules.

File: client/js/screens/Login/styles.js (hunk 2 above)
Comment (agalcalledjen) [reply client/js/screens/Login/styles.js]: Removed width and replaced it with flexbox.

**Pull request closed without merging**